### PR TITLE
Fix configmap docs

### PR DESCRIPTION
**Pull request by janetkuo**
There were a bunch of errors in the docs which made it hard to follow the example. 

@kubernetes/docs @pwittrock @pmorie

### CODE REVIEW

**Comment by janetkuo** [comment]
@johndmulhausen is docs/user-guide/configmap/README.md on the website?

**Comment by bgrant0607** [comment]
Is this another case of URL collision?

http://kubernetes.io/docs/user-guide/configmap/

**Comment by bgrant0607** [comment]
@pmorie It would be useful to dedupe these docs at some point.  LGTM. Merging.
